enterprise/server/api/api_server.go

```diff
@@ -103,6 +104,10 @@ func (s *APIServer) GetInvocation(ctx context.Context, req *apipb.GetInvocationR
 
 	invocations := []*apipb.Invocation{}
 	err = db.ScanEach(rq, func(ctx context.Context, ti *tables.Invocation) error {
+		bazelExitCode := ti.BazelExitCode
+		if ti.InvocationStatus == int64(invocation_status.InvocationStatus_DISCONNECTED_INVOCATION_STATUS) {
+			bazelExitCode = "disconnected"
+		}
```

Comment:
wdyt about adding invocation status to the returned object rather than adding logic here in the api layer (which is mostly just copying to / from our internal stuff)?

Reply:
I considered that too, but I think it would be most helpful as a string and not an int, which would require some translation logic here anyway. I figured it was simpler to reuse an existing field which also has the benefit of having richer, more detailed information

Reply:
I agree with Tyler's comment - storing "disconnected" in `bazelExitCode` doesn't "type-check" because the invocation BES status (PARTIAL/DISCONNECTED/COMPLETED) is representing something different than the bazel exit code.

re `string` vs `int` I think probably a proto enum is more appropriate? (we might have to copy the private InvocationStatus enum to the API proto definition). enums will appear as strings in the JSON-marshaled object, e.g. `Status: inspb.InvocationStatus_DISCONNECTED_INVOCATION_STATUS` in Go will appear as `"status": "DISCONNECTED_INVOCATION_STATUS"` in JSON I think.

Reply:
Okay that's fair. Changed